src/localization/localization.ts

```diff
@@ -128,7 +129,19 @@ const localization = {
             join: "Join",
             connected: (count: number) => `Connected to ${pluralize(count, "Zeplin Component")}.`,
             connect: "Connect to Zeplin components",
-            notFound: (name: string) => `No component matching “${name}” in projects/styleguides you’re a member of`,
+            askMigration: "Would you like to migrate zeplinNames on your configuration to zeplinIds?",
```

Comment:
```suggestion
            askMigration: "Would you like to migrate zeplinNames on the configuration file to zeplinIds?",
```

Reply:
"file" is okey, but for "the" see my other comments